src/core/include/iDynTree/Core/SpatialInertiaRaw.h

```diff
@@ -31,9 +31,9 @@ namespace iDynTree
     class SpatialInertiaRaw
     {
     protected:
-        double m_mass; /** mass */
-        double m_mcom[3]; /** first moment of mass (i.e. mass * center of mass */
-        RotationalInertiaRaw m_rotInertia; /** rotational inertia */
+        double m_mass; /** Mass. */
```

Comment:
These aside comments are passed on as documentation to the next variable declaration. For instance, "Mass" is documented in m_mcom[3].

Reply:
Use `///< My comment`